addons/compat_rhs_afrf3/compat_rhs_afrf3_csw/config.cpp

```diff
@@ -0,0 +1,45 @@
+#include "script_component.hpp"
+
+#if __has_include("\rhsafrf\addons\rhs_main\loadorder\config.bin")
+#else
+    #define PATCH_SKIP "RHS AFRF"
+#endif
+
+#if __has_include("\z\ace\addons\csw\script_component.hpp")
+#else
+    #ifdef PATCH_SKIP
+    #else
+        #define PATCH_SKIP "ACE Crew Served Weapons"
+    #endif
+#endif
+
+#ifdef PATCH_SKIP
+ACE_PATCH_NOT_LOADED(ADDON,PATCH_SKIP)
+#else
+
+#include "\z\ace\addons\csw\script_config_macros_csw.hpp"
```

Comment:
I think this will have to go between line 8 and 9?

Reply:
Does it matter? Moved anyway.